### PR TITLE
Add information about go-criu's version and release scheme

### PR DESCRIPTION
**Pull request by adrianreber**
Signed-off-by: Adrian Reber <areber@redhat.com>

### CODE REVIEW

**Comment by adrianreber** [comment]
Related to issue #5

File: README.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ or to just check if at least a certain CRIU version is installed:
 	result, err := c.IsCriuAtLeast(31100)
 ```
 
+## Releases
+
+go-criu will carry the same version number as
+[CRIU](https://github.com/checkpoint-restore/criu). Having the same version
+numbers as CRIU also means that each go-criu release will pull in the
+necessary changes from CRIU before making a release.
+
```
Comment (rst0git) [inline README.md]: Although the protocol buffers guarantees backwards compatibility, should we mention that the go-criu is for version 3.11 onwards?

Could we avoid the repetition of _"the same version numbers as CRIU"_? For example with:
`s/Having the same version numbers as CRIU also means that/This implies that/g`

Comment (adrianreber) [reply README.md]: Changed and force pushed.

**Comment by rst0git** [comment]
LGTM

**Pull request merged**